### ISSUE TITLE
Compatibility issue with Telemachus 1.4.28.0

### ISSUE THREAD

**Issue by johnwayne1930** [chrisnic/mkon#8]
Telemachus was today updated to version 1.4.28.0 for KSP 1.0.2.
http://forum.kerbalspaceprogram.com/threads/24594

Since that it seems MkON doesn't get connection to the Telemachus plugin.

**Comment by benbob11** [comment]
Me and my friend play Telemachus online and we wanted to try out a better skin. We found Mkon and instantly liked it however, since we're using 1.0.2, it doesn't work. Do have any idea how long it's going to take to fix?

**Comment by chrisnic** [comment]
Hey, thanks for letting me know about this. I haven't done much work on MKON since last year but i'll take a look over the next few days and see if I can get it running again for you.

**Comment by benbob11** [comment]
Is there any luck with the connection problem yet?

**Comment by phwoelfel** [comment]
This is probably more on the telemachus side!
I enabled debug for mkon and the data received from telemachus is the following:
```
Received Data{:"Mark1-2Pod","v.name":1,"p.paused":"1.4.28.0"}
```
Which is a faulty JSON string which causes the JSON parser to throw an error:
```
[Error] SyntaxError: JSON Parse error: Expected '}'
	parse (jquery-2.1.0.min.js, line 4)
	parseJSON (jquery-2.1.0.min.js, line 4)
	filterData (main.min.js, line 1)
	onmessage (main.min.js, line 1)
```

**Comment by lawnmowerlatte** [comment]
@phwoelfel, update Telemachus to [1.4.29.0](https://github.com/richardbunt/Telemachus/releases/tag/v1.4.29.0) — this fixes [issue #35](https://github.com/richardbunt/Telemachus/issues/35)

**Comment by phwoelfel** [comment]
It was indeed fixed in 1.4.29.0 but the uploads on all the platforms were faulty.
